src/bilder/images/concourse/deploy.py

```diff
@@ -133,8 +133,7 @@
         enable_p2p_volume_streaming=True,
         prometheus_bind_ip=IPv4Address("127.0.0.1"),
         prometheus_bind_port=CONCOURSE_PROMETHEUS_EXPORTER_DEFAULT_PORT,
-        secret_cache_duration="5m",  # pragma: allowlist secret
-        secret_cache_enabled=True,
+        secret_cache_enabled=False,  # Go to vault every time for secrets
```

Comment:
What's the motivation here? I know that we had experienced performance issues related to OCW pipelines fetching the SSH credentials for git previously.

Reply:
It is annoying. Mostly just frustrations around adjusting the tokens and it _seemed_ to hold on to the historical version of the token for much longer than 5 minutes. Maybe I can try just 1m and see if that is less obnoxious.